frontend/src/components/RequestDialog.vue

```diff
@@ -14,47 +8,36 @@
       :actions="false"
       @submit="submitRequestTransaction"
     >
-      <Card class="bg-teal px-20 pt-18 pb-16 self-stretch mb-11">
-        <RequestFormInputs v-if="requestState === RequestState.Init" />
-        <RequestStatus v-else :metadata="requestMetadata!" :state="requestState" />
-        <Transition name="expand">
-          <div v-if="shownError" class="mt-7 text-right text-lg text-orange-dark">
-            {{ shownError }}
-          </div>
-        </Transition>
-      </Card>
-
-      <div v-if="!ethereumProvider.signer">
-        <FormKit
-          input-class="w-112 bg-orange flex flex-row justify-center"
-          type="button"
-          @click="runRequestSigner"
-        >
-          <div v-if="requestSignerActive" class="h-8 w-8">
-            <spinner></spinner>
-          </div>
-          <template v-else>Connect MetaMask Wallet</template>
-        </FormKit>
-      </div>
-      <div v-else>
-        <FormKit
-          v-if="requestState === RequestState.Init"
-          class="w-72 flex flex-row justify-center bg-green"
-          type="submit"
-          :disabled="!valid"
-        >
-          Transfer funds
-        </FormKit>
-
-        <FormKit
-          v-if="requestState !== RequestState.Init"
-          input-class="w-72 flex flex-row justify-center bg-green"
-          type="button"
-          :disabled="isNewTransferDisabled"
-          @click="newTransfer"
-          >New Transfer</FormKit
-        >
-      </div>
+      <RequestFormInputs v-if="requestState === RequestState.Init" />
+      <RequestStatus v-else :metadata="requestMetadata!" :state="requestState" />
+      <Transition name="expand">
+        <div v-if="transferError" class="mt-7 text-right text-lg text-orange-dark">
+          {{ transferError }}
+        </div>
+      </Transition>
+
+      <Teleport to="#action-button-portal">
```

Comment:
I have red flags in my IDE for using Teleport and FormKit components without importing them in `script` part!

Reply:
If your IDE complains I'll fix that ofc. 😉

Reply:
Okay, so `FormKit` is actually already imported... And if I import `Teleport` from `vue`, linting actually complains that it is unused and makes the CI fail.
Can you confirm that?

Reply:
mmm.. I have tried to import it and the IDE linting had the same complaint! :thinking:  .. Not sure what to do actually so satisfy "Master Linting"! :)

Reply:
OK.. all is good.. It's this stupid IDE which became insane recently!